src/exportable/Exportable.ts

```diff
@@ -1,3 +1,18 @@
+import { ExportableCodeSystem } from './ExportableCodeSystem';
+import { ExportableExtension } from './ExportableExtension';
+import { ExportableInvariant } from './ExportableInvariant';
+import { ExportableMapping } from './ExportableMapping';
+import { ExportableProfile } from './ExportableProfile';
+import { ExportableValueSet } from './ExportableValueSet';
+
 export interface Exportable {
   toFSH(): string;
 }
+
+export type NamedExportable =
+  | ExportableProfile
+  | ExportableExtension
+  | ExportableValueSet
+  | ExportableCodeSystem
+  | ExportableInvariant
+  | ExportableMapping;
```

Comment:
Is there a reason `ExportableInstance` is not considered a `NamedExportable`?

Reply:
I'm sure there is a reason, but likely not a good one!